x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_embeddable.tsx

```diff
@@ -28,16 +28,25 @@ import { getRequestsAndResponses } from './utils';
 import { SourcererScopeName } from '../../store/sourcerer/model';
 import { VisualizationActions } from './actions';
 
+const HOVER_ACTIONS_PADDING = 24;
+
 const LensComponentWrapper = styled.div<{
-  height?: string;
-  width?: string;
+  $height?: number;
+  width?: string | number;
   $addHoverActionsPadding?: boolean;
 }>`
-  height: ${({ height }) => height ?? 'auto'};
+  height: ${({ $height }) => `${$height}px` ?? 'auto'};
```

Comment:
Consider revisiting the `??` in this expression, because it no longer returns `auto` when `$height` is `undefined`

Reply:
Thanks so much for catching this Andrew!